media_driver/linux/common/ddi/media_libva.cpp

```diff
@@ -1398,12 +1398,15 @@ VAStatus DdiMedia__Initialize (
     output_dri_init(ctx);
 #endif
 
+    void *gmm_handle = dlopen(GMM_UMD_DLL, RTLD_NOW);
```

Comment:
You _link_ against igdgmm, you don't need to dlopen() it and search for symbols.